### PR TITLE
Tip 3 Tweaks

### PR DESCRIPTION
**Pull request by rasbt**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

This tip is pretty succinct and spot on. I don't have any major suggestions besides some minor tweaks.

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]
LGTM. Thanks!

File: content/05.dl-complexities.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-To get good results, you must expect to train many networks across a range of different parameter and hyperparameter settings.
-Such training can be made more difficult by the demanding nature of these deep networks, which often require extensive computing infrastructure and optimization in order to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
-Furthermore, this experimentation is often noisy, which necessitates increased repetition and exacerbates the organizational challenges inherent to deep learning.
-On the whole, all code, random seeds, parameters, and results must be carefully corralled using general good coding practices (e.g. version control [@doi:10.1371/journal.pcbi.1004947], continuous integration etc.) in order to be effective and interpretable.
-This organization is also fundamental to the efficient sharing and reproducibility of research work [@doi:10.1371/journal.pcbi.1003285; @arxiv:1810.08055], and to the ability to keep models up to date as new data becomes available.
+To get good results, you must expect to train many networks across a wide range of different hyperparameter settings.
```
Comment (cgreene) [inline content/05.dl-complexities.md:4]:
```suggestion
To get good results, one must often train networks across a wide range of different hyperparameter settings.
```

File: content/05.dl-complexities.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+Such training can be made more difficult by the demanding nature of these deep networks, which often require extensive time investments into tuning and computing infrastructure to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
+Furthermore, this experimentation is often noisy, necessitating increased repetition and exacerbating the organizational challenges inherent to deep learning.
+On the whole, all code, random seeds, parameters, and results must be carefully corralled using general coding standards and best practices (for example, version control [@doi:10.1371/journal.pcbi.1004947] and continuous integration) to be reproducible and interpretable [@doi:10.1371/journal.pcbi.1003285; @arxiv:1810.08055].
```
Comment (cgreene) [inline content/05.dl-complexities.md:7]:
```suggestion
On the whole, all code, random seeds, parameters, and results must be carefully corralled using general coding standards and best practices (for example, version control [@doi:10.1371/journal.pcbi.1004947] and continuous integration [@doi:10.1038/nbt.3780]) to be reproducible and interpretable [@doi:10.1371/journal.pcbi.1003285; @arxiv:1810.08055].
```

File: content/05.dl-complexities.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 That is, the CUDA/CuDNN architectures that facilitate the parallelized computing that power state-of-the-art DL often use algorithms by default that produce different outcomes from iteration to iteration.
-Therefore, achieving reproducibility in this context requires explicitly specifying the use of deterministic algorithms (which are typically available within your deep learning library), which is distinct from the setting of random seeds that typically achieve reprocubility by controlling pseudorandom deterministic procedures such as shuffling and initialization [@url:https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#reproducibility].
+Therefore, achieving reproducibility in this context requires explicitly specifying the use of deterministic algorithms (which are typically available within your deep learning library), which is distinct from the setting of random seeds that typically achieve reproducibility by controlling pseudorandom deterministic procedures such as shuffling and initialization [@url:https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#reproducibility].
```
Comment (cgreene) [inline content/05.dl-complexities.md:12]:
```suggestion
Therefore, achieving reproducibility in this context requires explicitly specifying the use of deterministic algorithms (which are typically available within deep learning libraries), which is distinct from the setting of random seeds that typically achieve reproducibility by controlling pseudorandom deterministic procedures such as shuffling and initialization [@url:https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#reproducibility].
```

File: content/05.dl-complexities.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-Again, beware of the choices that are being made implicitly (that is, by default settings) by your framework of choice (for example, choice of optimization algorithm), as these seemingly trivial specifics can actually have signficant effects on model performance.
-For example, adaptive methods often lead to faster convergence during training, but may lead to worse generalization performance on independent datasets [@url:https://papers.nips.cc/paper/7003-the-marginal-value-of-adaptive-gradient-methods-in-machine-learning]).
-These nuanced elements are easy to overlook, but it is critical to carefully consider them and to evaluate their potential impact (see [Tip 6](#hyperparameters)).
+This can help prevent practitioners from wasting significant time and resources on running highly complex models that feature numerous unresolved problems. 
+Again, beware of the choices made implicitly (that is, by default settings) by your framework of choice (for example, selection of optimization algorithm), as these seemingly trivial specifics can have significant effects on model performance.
```
Comment (cgreene) [inline content/05.dl-complexities.md:16]:
```suggestion
Again, beware of the choices made implicitly (that is, by default settings) by deep learning libraries (for example, selection of optimization algorithm), as these seemingly trivial specifics can have significant effects on model performance.
```

**Review by cgreene — APPROVED** [review]
eliminated second person while doing the read through 🚀

**Pull request merged**